tensorflow/python/debug/cli/analyzer_cli_test.py

```diff
@@ -46,6 +46,11 @@
 from tensorflow.python.platform import test
 from tensorflow.python.util import tf_inspect
 
+# Helper function to accommodate MKL-enabled TensorFlow:
+# MatMul op is supported by MKL and its name is prefixed with "_Mkl" during the
+# MKL graph rewrite pass.
+def matmul_op_name():
```

Comment:
Nit: matmul_op_name() --> _matmul_op_name()

Reply:
Incorporated the change in https://github.com/tensorflow/tensorflow/commit/9b3f1d992a96c626cb39e5199ebcce3c64e89e1b. Resolving this comment.